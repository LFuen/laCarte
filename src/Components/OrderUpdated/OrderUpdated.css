.flexed {
    display: flex;
    justify-content: center;
    align-items: center;
    margin-top: 25px;
}

.flexedMid {
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
}