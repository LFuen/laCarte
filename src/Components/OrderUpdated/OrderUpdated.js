import React, {Component} from 'react'
import {Link} from 'react-router-dom'
import '../OrderUpdated/OrderUpdated.css'


class OrderUpdated extends Component{

    render(){
        return(
            <div className='flexed'>
                <span title='Received' className='flexedMid'>
                    <img className='order' alt='orderReceived' src='https://lfuen.github.io/laCarte/images/mealUpdated.png'/>
                </span>    
                <Link to='/OrderHistory'><button type='button' className='shadow'>View Orders</button></Link>
            </div>
        )
    }
}


export default OrderUpdated