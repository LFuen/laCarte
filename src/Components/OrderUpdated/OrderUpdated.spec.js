import React from 'react'
import ReactDOM from 'react-dom'
import { BrowserRouter } from 'react-router-dom'
import LaCarteContext from '../../context/LaCarteContext'
import OrderUpdated from './OrderUpdated'



describe(`OrderUpdated Component`, () => {
    describe(`Smoke test`, () => {
        it(`Renders without crashing`, () => {
            
            const div = document.createElement('div');

            const value = {
                meals: 'string',
                chefs: 'string',
                cuisines: 'string',
                orders: 'string',
                addOrder: 'string',
                updateOrder: 'string',
                deleteOrder: 'string',
            };

            ReactDOM.render(
                <BrowserRouter>
                    <LaCarteContext.Provider value={value}>
                        <OrderUpdated />
                    </LaCarteContext.Provider>                
                </BrowserRouter>,
                div)
            ReactDOM.unmountComponentAtNode(div)
        })
    })
})