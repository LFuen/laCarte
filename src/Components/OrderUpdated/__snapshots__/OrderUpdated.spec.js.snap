// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrderUpdated Component Snapshot renders the UI as expected 1`] = `
<div
  className="flexed"
>
  <span
    className="flexedMid"
    title="Received"
  >
    <img
      alt="orderReceived"
      className="order"
      src="https://lfuen.github.io/laCarte/images/mealUpdated.png"
    />
  </span>
  <a
    href="/OrderHistory"
    onClick={[Function]}
  >
    <button
      className="shadow"
      type="button"
    >
      View Orders
    </button>
  </a>
</div>
`;
